src/fcmp_pp/fcmp_pp_rust/fcmp++.h

```diff
@@ -219,6 +221,10 @@ CResult output_blinds_new(const uint8_t *o_blind,
 CResult helios_branch_blind(void);
 CResult selene_branch_blind(void);
 
+int new_helios_branch_blind(struct HeliosBranchBlindUnsafe **branch_blind_out);
```

Comment:
I would prefer a term like `generate_helios_branch_blind`, which is a bit more descriptive than "new", but again, personal choice.